src/core/connected-graph/hierarchy.ts

```diff
@@ -572,7 +572,7 @@ export class D3HierarchyGraph extends D3ConnectedGraphSVG {
             let contentStyle = getStyle(nodeContents);
             let xOffset = (rect.width - nodeContentRect.width) / 2;
             let yPadding = defaultPadding;
-            let yOffset = yPadding;
+            let yOffset = yPadding + topRect.height;
```

Comment:
@tonym97,  you and I agree on this change, but just to document the reason, we want the nodeContents to appear below the topRect.